Products/ZenRelations/PrimaryPathObjectManager.py

```diff
@@ -65,7 +65,7 @@ def getPrimaryId(self, fromNode=None):
     def getPrimaryUrlPath(self, full=False):
         """Return the primary path as an absolute url"""
         objaq = self.primaryAq()
-        if full: return objaq.absolute_url()
+        if full: return objaq.absolute_url_path()
         return objaq.absolute_url_path()
```

Comment:
I think, in this case, the existing code is fine.  The `absolute_url()` method is only called if the `full` parameter is passed a `True` argument.